doc/org/build/networking.md

```diff
@@ -0,0 +1,256 @@
+
+## `CreatorChange`
+
+These tell a client that the `creator` of an entity has been reassigned. Typically, this means that an entity has been `pinned` (or `unpinned`), and reticulum has assigned (or unassigned) itself as the entity&rsquo;s `creator`.
+
+
+<a id="orga8173d0"></a>
+
+# Eventual Consistency
+
+Reticulum does not enforce a single, consistent networked entity state. In fact, reticulum knows very little about the messages it is passing between clients. Furthermore, messages are not guaranteed to be received in the same order by all clients. Therefore, it is each client&rsquo;s responsibility to handle messages in such a way that all clients will eventually recreate identical entity state. This general concept is called eventual consistency.
+
+Most of the complexity in the `networkSendSystem` and the `networkReceiveSystem` stem from this property of the network. Here are some examples where this complexity reveals itself:
+
+-   The `lastOwnerTime` is used to ensure that ownership transfer is handled identically by all clients, even when messages arrive out of order.
+-   The `deletedNids` collection ensures that out-of-order `CreateMessage` s do not cause `deleted` entities to be accidentally recreated.
+-   The `storedUpdates` allows a client to save `UpdateMessage` s it has received but has no way to process, as can happen when it receives an `UpdateMessage` from the `owner` of an entity before it receives a `CreateMessage` from its `creator`.
+-   The `takeSoftOwnership` function allows clients to take ownership of unowned entities in such a way that only clients with the most recent information about that entity will be eligible as the new owner.
+
+For the most part, users of the networking systems do not need to understand these concepts. These are handled internally by the systems themselves. However, users do need to understand that ownership is not transactional or guaranteed. That is, ownership is not &ldquo;requested and then transferred&rdquo;, and just because one client claims ownership of an entity does not mean that other clients will respect that claim.
+
+Users can inspect the state the `Networked` or `Owned` components as needed in cases when their ownership claims matter. They may find themselves writing coroutines that looks like this:
+
+```typescript
+takeSoftOwnership(world, eid);
+yield sleep( 3000 ); // Wait a few seconds to see if we "win" ownership
+if (!hasComponent(world, Owned, eid)) return;
+```
+
+If this becomes a common and error-prone pattern, then we may introduce helper functions or additional semantics to cover these cases.
+
+
+<a id="org62cc95d"></a>
+
+# Creating Networked Entities
+
+Client code creates networked entities by calling `createNetworkedEntity`, passing:
+
+-   A `prefabName`, to indicate which prefab to initialize,
+-   An `initialData` struct, to know how to initialize the prefab.
+
+Prefabs must be registered in `prefabs` , a map from `PrefabName` to `PrefabDefinition`.
```

Comment:
Example code of prefab registration and/or link to `src/prefabs/prefabs.ts` where built-in prefabs are registered in the Hubs core would be helpful to understand.